atomic_reactor/plugins/pre_fetch_maven_artifacts.py

```diff
@@ -147,8 +153,10 @@ def process_by_nvr(self, nvr_requests):
                 # not contain a trailing slash, which causes the last dir to
                 # be dropped.
                 url = maven_build_path + '/' + maven_file_path
+                checksum_type = koji.CHECKSUM_TYPES[build_archive['checksum_type']]
```

Comment:
`koji.CHECKSUM_TYPES` is used to convert the value we get from koji into something meaningful. Koji reports the checksum_type as 0, we use `koji.CHECKSUM_TYPES` to get the string `'md5'` from it.

The `CHECKSUM_TYPES` dict defined in this plugin has a different purposed. It maps names of hashing algorithms  to the actual hashing function: `'md5' => haslib.md5`